angular.json

```diff
@@ -60,9 +86,21 @@
                   "with": "src/environments/environment.prod.ts"
                 }
               ]
+            },
+            "en": {
+              "localize": ["en"],
+              "baseHref": "/en/"
+            },
+            "fr": {
+              "localize": ["fr"],
+              "baseHref": "/fr/"
+            },
+            "sv": {
+              "localize": ["sv"],
+              "baseHref": "/sv/"
             }
```

Comment:
Why is this limited to three out of seven languages?

Reply:
No I forgot to add the other, fixed